xml/net/protocol.xml

```diff
@@ -111,6 +124,7 @@
     </struct>
 
     <struct name="PaperdollBAHSW">
+        <comment>Player equipment data</comment>
```

Comment:
Should you describe the different structs here? Like "Player equipment data (only boots, armor, hat, sword, shield)" ?

Reply:
Decided against adding a top-level comment describing the fields in that manner, but I ended up adding an extra comment to mention that these were graphic IDs rather than item IDs.

Reply:
Actually, I will rename the structs and add comments indicating how they're used.
I think `Paperdoll` is the wrong name.